architecture-building-systems/CityEnergyAnalyst
# I1099 change temperature outputs in sc total and pvt total

This PR resolves #1099 
The temperature outputs in the total files are aggregated, which is changed into average in this branch.

changes in this PR:
- In `solar_collector.py` and `photovoltaic_thermal.py`
- New outputs in `SC_total.csv` and `PVT_total.csv`: The temperatures are average instead of sum now.
- Connect inputs from `cea.config`
- Delete `settings.py`, add `constants.py`
- Refactoring

to test:

- run `solar_collector.py` and `photovoltaic_thermal.py` , open `SC_total.csv` and `PVT_total.csv`, and check columns (`T_SC/PVT_sup_C`&`T_SC/PVT_re_C`)
